components/core/UnityFrame.js

```diff
@@ -46,7 +46,11 @@ const UnityFrame = ({ url, unityGameLoader, unityGameConfig }) => {
     };
   }, []);
 
-  return <div id="unityContainer" css={STYLES_CONTAINER} />;
+  return (
+    <div css={STYLES_CONTAINER} onClick={(e) => e.stopPropagation()}>
```

Comment:
I think this is okay for now, there is a better way I'll teach some other time involving a traversal up the tree and some domain knowledge around bubbling and capture.